v2.0/check.md

```diff
@@ -4,32 +4,33 @@ summary: The Check constraint specifies that values for the column in INSERT or
 toc: false
 ---
 
-The Check [constraint](constraints.html) specifies that values for the column in [`INSERT`](insert.html) or [`UPDATE`](update.html) statements must return `TRUE` or `NULL` for a Boolean expression. If any values return `FALSE`, the entire statement is rejected.
+The `CHECK` [constraint](constraints.html) specifies that values for the column in [`INSERT`](insert.html) or [`UPDATE`](update.html) statements must return `TRUE` or `NULL` for a Boolean expression. If any values return `FALSE`, the entire statement is rejected.
 
 <div id="toc"></div>
 
 ## Details
 
-- If you add a Check constraint to an existing table, existing values are not checked. However, any updates to those values will be.
-  {{site.data.alerts.callout_info}}In the future we plan to expand the Check constraint to include a check on any existing values in the column.{{site.data.alerts.end}}
-- Check constraints may be specified at the column or table level and can reference other columns within the table. Internally, all column-level Check constraints are converted to table-level constraints so they can be handled consistently.
-- You can have multiple Check constraints on a single column but ideally, for performance optimization, these should be combined using the logical operators. For example
+- If you add a `CHECK` constraint to an existing table, existing values are not checked. However, any updates to those values will be.
+  {{site.data.alerts.callout_info}}In the future we plan to expand the `CHECK` constraint to include a check on any existing values in the column.{{site.data.alerts.end}}
```

Comment:
This should link to the github issue where progress is tracked.

Reply:
I found [#8773](https://github.com/cockroachdb/cockroach/issues/8773) but can't find the specific issue for CHECK constraint. Do you know what issue is used to track this?

Reply:
https://github.com/cockroachdb/cockroach/issues/23663

Reply:
👍 thanks!